config/config.go

```diff
@@ -776,3 +782,39 @@ func (re Regexp) MarshalJSON() ([]byte, error) {
 	}
 	return nil, nil
 }
+
+// Matchers is label.Matchers with an added UnmarshalYAML method to implement the yaml.Unmarshaler interface.
+type Matchers labels.Matchers
+
+// UnmarshalYAML implements the yaml.Unmarshaler interface for Matchers.
+func (m *Matchers) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var lines []string
+	if err := unmarshal(&lines); err != nil {
+		return err
+	}
+	for _, line := range lines {
+		pm, err := labels.ParseMatchers(line)
+		if err != nil {
+			return err
+		}
+		*m = append(*m, pm...)
+	}
+	sort.Sort(labels.Matchers(*m))
+	return nil
+}
+
+// MarshalYAML implements the yaml.Marshaler interface for Matchers.
+func (m *Matchers) MarshalYAML() (interface{}, error) {
+	var buf bytes.Buffer
+
+	buf.WriteByte('{')
+	for i, ms := range *m {
```

Comment:
`ms` is kind of misleading, sounds like the plural of m. I would either use `ms` as the name for the `Matchers` receiver (but then needs to be changed throughout all methods of `Matchers`), or use something else here, e.g. `matcher`.